ktor-http/common/src/io/ktor/http/HttpUrlEncoded.kt

```diff
@@ -29,25 +41,29 @@ fun String.parseUrlEncodedParameters(defaultEncoding: Charset = Charsets.UTF_8,
 }
 
 /**
- * Encode form parameters from a list of pairs
+ * Encode form parameters from a list of pairs by using [option]
  */
-fun List<Pair<String, String?>>.formUrlEncode(): String = StringBuilder().apply {
```

Comment:
Breaking ABI change, please keep original function and annotate it with `@Deprecated` with hidden visibility to fix that.